go.mod

```diff
@@ -21,150 +21,160 @@ require (
 	github.com/open-policy-agent/gatekeeper/v3 v3.13.0
 	github.com/openshift-kni/k8sreporter v1.0.4
 	github.com/openshift-psap/special-resource-operator v0.0.0-00010101000000-000000000000
-	github.com/openshift/api v3.9.0+incompatible
-	github.com/openshift/client-go v3.9.0+incompatible
+	github.com/openshift/api v0.0.0-20230807121159-a81c3efc8824
```

Comment:
while bumping, can you please go to the latest openshift/api?

Reply:
sure, done!